scenario/3_struct_mapping_extend.yml

```diff
@@ -0,0 +1,37 @@
+input:
+    input.go: |
+        package structs
```

Comment:
Is your plan to incrementally update this PR with all the features?

Reply:
Yes. I wanted to sketch out the design first, see if we're on the same page 😄